### PR TITLE
feat: Add Portuguese language

### PR DESCRIPTION
**Pull request by qlereboursBS**
## Describe your changes

I added the Portuguese language. However it has been translated with ChatGPT and it needs to be validated because I don't speak a word of Portuguese 😄
For instance, the "Admin" nav stays "Admin" but I'm not sure if it should be the case

Also, I saw something that may be a mistake: The text content of the DemoLoginHint component (This is a read-only demo, but you can log in with admin/admin ......) is hard coded, even though it's also translated in the translation files. Is it like this on purpose?

## Screenshots

image
image
image
image

## Documentation

Nothing to add in the documentation.

## Checklist

 - [x] I performed a self review of my code
 - [x] I ensured that everything is written in English
 - [x] I tested the feature or fix on my local environment
 - [x] I ran the `yarn storybook` command and everything is working
 - [x] If applicable, I updated the translations for english and french files  
      (If you cannot update the french language, just let us know in the PR description)
 - [x] If applicable, I updated the README.md
 - [x] If applicable, I created a PR or an issue on the [documentation repository](https://github.com/bearstudio/start-ui-web-docs/)
 - [x] If applicable, I’m sure that my feature or my component is mobile first and available correctly on desktop

### CODE REVIEW

**Review by ivan-dalmet — APPROVED** [review]

**Comment by ivan-dalmet** [comment]
For the hard coded text of DemoLoginHint, it's normal that it's hard coded but there should not be the keys in the translations, you're right. If you can remove the unused keys for that 🙏

**Comment by qlereboursBS** [comment]
Ok @ivan-dalmet I just removed the translation file because any translation was used

**Comment by taorepoara** [comment]
On which version should I base the review, the French one or the English one ?

File: src/locales/pt/users.json [hunk 1]
```diff
@@ -0,0 +1,109 @@
+{
+  "list": {
+    "title": "Gerenciamento de Usuários",
+    "actions": {
+      "createUser": "Criar Usuário"
```
Comment (taorepoara) [inline src/locales/pt/users.json:5]:
```suggestion
    "title": "Gerenciamento de usuários",
    "actions": {
      "createUser": "Criar usuário"
```

File: src/locales/pt/users.json [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    }
+  },
+  "feedbacks": {
+    "activateUserSuccess": {
+      "title": "Usuário Ativado",
```
Comment (taorepoara) [inline src/locales/pt/users.json:10]:
```suggestion
      "title": "Usuário ativado",
```

File: src/locales/pt/users.json [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+      "description": "Usuário \"{{login}}\" ativado com sucesso"
+    },
+    "deactivateUserSuccess": {
+      "title": "Usuário Desativado",
```
Comment (taorepoara) [inline src/locales/pt/users.json:14]:
```suggestion
      "title": "Usuário desativado",
```

File: src/locales/pt/users.json [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+    "activateUserError": {
+      "title": "Falha na Ativação",
+      "description": "Falha ao ativar o usuário \"{{login}}\""
+    },
+    "deactivateUserError": {
+      "title": "Falha na Desativação",
+      "description": "Falha ao desativar o usuário \"{{login}}\""
+    },
+    "deleteUserSuccess": {
+      "title": "Usuário Excluído",
```
Comment (taorepoara) [inline src/locales/pt/users.json:26]:
```suggestion
      "title": "Usuário excluído",
```

File: src/locales/pt/users.json [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+      "description": "Falha ao remover o usuário \"{{login}}\""
+    },
+    "loadingUserError": {
+      "title": "Falha ao carregar usuários"
+    }
+  },
+  "data": {
+    "login": {
+      "alreadyUsed": "Nome de usuário já utilizado",
+      "label": "Nome de Usuário",
```
Comment (taorepoara) [inline src/locales/pt/users.json:40]:
```suggestion
      "label": "Nome de usuário",
```

File: src/locales/pt/users.json [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+      "required": "Nome de usuário é obrigatório",
+      "tooShort": "Nome de usuário é muito curto (mín. {{min}} caracteres)",
+      "tooLong": "Nome de usuário é muito longo (máx. {{max}} caracteres)",
+      "invalid": "Nome de usuário é inválido, não use caracteres especiais"
```
Comment (taorepoara) [inline src/locales/pt/users.json:44]:
```suggestion
      "required": "O nome de usuário é obrigatório",
      "tooShort": "O nome de usuário é muito curto (mín. {{min}} caracteres)",
      "tooLong": "O nome de usuário é muito longo (máx. {{max}} caracteres)",
      "invalid": "O nome de usuário é inválido, não use caracteres especiais"
```

File: src/locales/pt/users.json [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+    },
+    "email": {
+      "label": "E-mail",
+      "alreadyUsed": "E-mail já utilizado",
+      "required": "E-mail é obrigatório",
+      "invalid": "E-mail é inválido",
+      "tooShort": "E-mail é muito curto (mín. {{min}} caracteres)",
+      "tooLong": "E-mail é muito longo (máx. {{max}} caracteres)"
```
Comment (taorepoara) [inline src/locales/pt/users.json:52]: There is diff here with the `account.json`. Both `email` and `e-mail` are used in Portuguese, you should choose one to use...

```suggestion
      "required": "O email é obrigatório",
      "invalid": "O email é inválido",
      "tooShort": "O email é muito curto (mín. {{min}} caracteres)",
      "tooLong": "O email é muito longo (máx. {{max}} caracteres)"
```

File: src/locales/pt/users.json [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+    },
+    "authorities": {
+      "label": "Autoridades",
+      "required": "Autoridades são obrigatórias"
+    },
+    "createdBy": {
+      "label": "Criado Por"
+    },
+    "modifiedBy": {
+      "label": "Modificado Por"
```
Comment (taorepoara) [inline src/locales/pt/users.json:65]:
```suggestion
      "label": "Modificado por"
```

File: src/locales/pt/users.json [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+    },
+    "status": {
+      "label": "Status",
+      "activated": "Ativado",
+      "deactivated": "Não Ativado"
+    },
+    "firstname": {
+      "label": "Primeiro Nome"
```
Comment (taorepoara) [inline src/locales/pt/users.json:73]:
```suggestion
      "label": "Nome"
```

File: src/locales/pt/users.json [hunk 10]
```diff
@@ -0,0 +1,109 @@
...
+    },
+    "lastname": {
+      "label": "Último Nome"
```
Comment (taorepoara) [inline src/locales/pt/users.json:76]:
```suggestion
      "label": "Sobrenome"
```

File: src/locales/pt/users.json [hunk 11]
```diff
@@ -0,0 +1,109 @@
...
+    },
+    "language": {
+      "label": "Idioma"
+    }
+  },
+  "update": {
+    "action": {
+      "save": "Atualizar Usuário"
```
Comment (taorepoara) [inline src/locales/pt/users.json:84]:
```suggestion
      "save": "Atualizar usuário"
```

File: src/locales/pt/users.json [hunk 12]
```diff
@@ -0,0 +1,109 @@
...
+      },
+      "updateSuccess": {
+        "title": "Atualizado com sucesso"
+      }
+    }
+  },
+  "create": {
+    "feedbacks": {
+      "updateError": {
+        "title": "Falha na Criação"
```
Comment (taorepoara) [inline src/locales/pt/users.json:98]:
```suggestion
        "title": "Falha na criação"
```

File: src/locales/pt/users.json [hunk 13]
```diff
@@ -0,0 +1,109 @@
...
+      },
+      "updateSuccess": {
+        "title": "Usuário criado com sucesso"
+      }
+    },
+    "action": {
+      "save": "Criar Usuário"
+    },
+    "title": "Criar Novo Usuário"
```
Comment (taorepoara) [inline src/locales/pt/users.json:107]:
```suggestion
      "save": "Criar usuário"
    },
    "title": "Criar novo usuário"
```

**Review by taorepoara — CHANGES_REQUESTED** [review]
Some changes needed

**Review by taorepoara — CHANGES_REQUESTED** [review]
Some changes needed

**Comment by yoannfleurydev** [comment]
Hey @qlereboursBS , since we released the "v2" of 🚀 Start UI [web], we'll need you to update this PR if possible. Thanks.

**Comment by qlereboursBS** [comment]
> Hey @qlereboursBS , since we released the "v2" of 🚀 Start UI [web], we'll need you to update this PR if possible. Thanks.

Ok, I plan to do it next week

**Pull request closed without merging**